Elemental/Components/AeTable/AeBaseTable.cs

```diff
@@ -38,6 +39,14 @@ protected virtual IEnumerable<T> GetSortedDataset()
             {
                 return new List<T>();
             }
+
+            if (DisableDefaultSort)
+            {
+                //The default sort should only be run when the table loads the first time.
+                DisableDefaultSort = false;
+                return Dataset;
+            }
+
```

Comment:
I think we should rename this to `DisableFirstSort` as that's a bit more explicit to what it's doing. I'm also good with adding it as a second parameter, ie. both `DisableFirstSort` and `DisableDefaultSort` - as long as the logic gets reworked to reflect the names.

Nice solution though, really interesting edge case!